scripts/repl.js

```diff
@@ -1,5 +1,15 @@
 (function(babel, $, _, ace, window) {
 
+  /* Throw meaningful errors for getters of commonjs. */
+  ["module", "exports", "require"].forEach(function(commonVar){
+    Object.defineProperty(window, commonVar, { 
+      get: function () { 
+      throw new Error(commonVar + " is not supported in the browser,\
+        you need a commonjs environment such as node.js/io.js, browserify/webpack etc");
```

Comment:
That's only included on the repl page and it would break CloudFlare's auto-minification I have setup right now. Come fix that when you start at CloudFlare and we'll talk :stuck_out_tongue_winking_eye: